packages/next-templates/src/app/globals.css

```diff
@@ -197,6 +198,44 @@ utrecht-alert
   align-self: center;
 }
 
+.voorbeeld-form-label {
+  display: block;
+}
+
+.voorbeeld-back-link {
+  display: flex;
+  gap: 5px;
```

Comment:
var(--voorbeeld-space-inline-snail) dan is het 6px maar je hebt dan een spacing token staan